ui/app/components/feedback/FeedbackTableModal.tsx

```diff
@@ -12,12 +12,19 @@ import {
 } from "~/components/layout/SnippetLayout";
 import { TextMessage } from "~/components/layout/SnippetContent";
 import { parseInferenceOutput } from "~/utils/clickhouse/inference";
-import { Output } from "~/components/inference/Output";
+import { ChatOutputElement } from "~/components/input_output/ChatOutputElement";
+import { JsonOutputElement } from "~/components/input_output/JsonOutputElement";
 
 interface FeedbackTableModalProps {
   feedback: FeedbackRow;
 }
 
+function isJsonOutput(
```

Comment:
this seems like a bad place to put this in the codebase. Can you move to `ui/app/utils/clickhouse/inference.ts`?